### DISCUSSION TITLE
Scene viewer fails the first time it loads.

### DISCUSSION THREAD

**Discussion by hd-esteban** [google/model-viewer#2828] [Q&A]
I am using Google Pixel 3 phone with Android version 11. Since I've already had this error on a regular Chrome browser, I used Incognito to test this. Paste the URL in incognito. Click on View in your space hyperlink. I received the error message "Unfortunately, your mobile device does not support this feature." with a product page button. Now, I hit back on the browser and basically refreshed the page again. I repeated the step of clicking View in your space hyperlink. This time AR appeared fine. Note that this error is shown only for the first time. The AR button is executing this intent link:

intent://arvr.google.com/scene-viewer/1.0?file=https%3A%2F%2Fcontent.cylindo.com%2Fapi%2Fv2%2F4768%2Fproducts%2F5002-10%2FAR%2FAR.glb%3Ffeature%3DBODY%3A959-80%26feature%3DLEG%2520FINISH%3AF&link=https%3A%2F%2Fflexsteel-dev.microdinc.com%2Fflexsteel-chair%2F5002-10-11%2Fiteminformation.aspx&title=3d%20Mode

the modal is shown when the intent link redirect to the fallback hash so for some reason Scene Viewer is locking the first execution of the link.

https://flexsteel-dev.microdinc.com/flexsteel-chair/5002-10-11/iteminformation.aspx

Scene viewer / Google Pixel 3 phone / Android version 11

### Browser Affected
- [ X] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ X] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [X ] SceneViewer
- [ ] QuickLook

https://user-images.githubusercontent.com/1881501/135001579-891d379d-cce1-4c8d-ba7c-cbdf7f4767a8.mov

**Comment by elalish** [comment]
@eugenecys Looks like a Scene Viewer regression?

**Comment by hd-esteban** [comment]
Hi @elalish / @eugenecys do you have any update on this? I'm able to replicate the issue using browser stack so it seems to be a problem with the engine of the browser/device

**Comment by elalish** [comment]
I can't repro in either Incognito or regular mode. Considering you don't get the "do you want  to leave incognito mode" prompt, it appears to be a  problem with Chrome failing the intent before it ever gets to  Scene Viewer. Feel free to  file a bug on  Chromium, though you  may need a more precise set of  repro steps.
